environments/pull-environments.sh

```diff
@@ -34,7 +34,7 @@ PROVIDERS="{\
 # Pull the backend for each environment
 for dir in ./*/ ; do
   if ! [ -f "$dir/package.json" ]; then
-    echo "If $dir is an environment, ensure the a package.json file exists with a \"pull\" command that pulls the environment (see the README)."
+    echo "If $dir is an environment, ensure the a package.json file exists with a \"dev\" command that pulls the environment (see the README)."
```

Comment:
Should this still be `pull`? https://github.com/aws-amplify/amplify-ui/blob/f0001b501747077ec7ae1f6ddf913f1391796532/environments/auth-with-multi-alias/package.json#L6

Reply:
Something automatically changed this. I'll change it back!